java/generate/templates/pom.mustache

```diff
@@ -446,6 +467,7 @@
         <swagger-annotations-version>1.5.21</swagger-annotations-version>
         <jersey-version>1.19.4</jersey-version>
         <jackson-version>2.12.1</jackson-version>
+        <jackson-databind-version>2.12.1</jackson-databind-version>
```

Comment:
What is this for?

Reply:
Nevermind I see that it was above. What was the reference of `${fasterxml-version}` used earlier and now this is hardcoded?

Reply:
Oh now I understand, okay new question, where was this version before?

Reply:
Ahh I see, you have seperated it from the jackson-version. Okay nice.